frontend/src/routes/Router.tsx

```diff
@@ -44,6 +46,15 @@ const routes = createRoutesFromElements(
         </Route>
       </Route>
 
+      <Route
+        path="/matches/:matchId"
+        element={
+          <SocketProvider>
+            <GameInterface />
+          </SocketProvider>
+        }
+      ></Route>
+
```

Comment:
Made necessary routing changes in a [separate pr](https://github.com/Koodikukkaro/Kendo-tournament-app/pull/65)